include/sway/criteria.h

```diff
@@ -66,7 +66,7 @@ void criteria_destroy(struct criteria *criteria);
 struct criteria *criteria_parse(char *raw, char **error);
 
 /**
- * Compile a list of criterias matching the given view.
+ * Compile a list of criterion matching the given view.
```

Comment:
I think this should be "criteria" actually, since it's a list of multiple criterion items. We say e.g. "list of strings" (plural form).